### PR TITLE
Add confirmation route and handler, relates to issue #11

### PR DESCRIPTION
**Pull request by RhodesPeter**
We have added the file confirmation.js which relates to issue #11

### CODE REVIEW

File: src/handlers/confirmation.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  }
+  const promise = yotiClient.getActivityDetails(token);
+  promise.then((activityDetails) => {
+    const userId = activityDetails.getUserId();
+    const profile = activityDetails.getUserProfile();
+    reply(profile);
+  })
+};
+
+const confirmationRoute = {
```
Comment (jsms90) [inline src/handlers/confirmation.js]: @RhodesPeter Do you want to be defining your routes inside a folder called "handlers"?

**Comment by RhodesPeter** [comment]
I have changed the file name from handlers to routes @jsms90.

**Comment by RhodesPeter** [comment]
Sorry I accidentally closed the PR. Ready to merge now :)

**Comment by jsms90** [comment]
@RhodesPeter Cool. You may want to separate your routes and handlers later. Modularisation is just an architectural choice (as far as I know). Just wanted to highlight it :+1:

**Pull request merged**